node_modules/oae-preview-processor/tests/test-previews.js

```diff
@@ -909,6 +909,71 @@ describe('Preview processor', function() {
+                .get('/youtube/v3/videos?part=snippet&id=lgTQ5I_H4Xk&key=')
+                .twice()
+                .reply(200, {
+                    'kind': 'youtube#videoListResponse',
+                    'etag': '\"tbWC5XrSXxe1WOAx6MK9z4hHSU8/CQhw2t_NZKBaw72WEH7b1hSa6RA\"',
+                    'pageInfo': {
+                        'totalResults': 1,
+                        'resultsPerPage': 1
+                    },
+                    'items': [
+                        {
+                            'kind': 'youtube#video',
+                            'etag': '"tbWC5XrSXxe1WOAx6MK9z4hHSU8/TScJmbzIomSQvSDFqAgTXGr7Y2U"',
+                            'id': 'lgTQ5I_H4Xk',
+                            'snippet': {
+                                'publishedAt': '2013-05-09T20:54:56.000Z',
+                                'channelId': 'UCzDbnWaP_5kd6HpvDUjoT4Q',
+                                'title': 'How to prounounce \"Apereo\"',
+                                'description': 'Here is Ian Dolphin, the Executive Director of the Apereo Foundation, with the official pronunciation of the word \"Apereo\".',
+                                'thumbnails': {
+                                    'default': {
+                                        'url': 'https://i.ytimg.com/vi/lgTQ5I_H4Xk/default.jpg',
+                                        'width': 120,
+                                        'height': 90
+                                    },
+                                    'medium': {
+                                        'url': 'https://i.ytimg.com/vi/lgTQ5I_H4Xk/mqdefault.jpg',
+                                        'width': 320,
+                                        'height': 180
+                                    },
+                                    'high': {
+                                        'url': 'https://i.ytimg.com/vi/lgTQ5I_H4Xk/hqdefault.jpg',
+                                        'width': 480,
+                                        'height': 360
+                                    }
+                                },
+                                'channelTitle': 'Apereo Foundation',
+                                'categoryId': '28',
+                                'liveBroadcastContent': 'none',
+                                'localized': {
+                                    'title': 'How to prounounce \"Apereo\"',
+                                    'description': 'Here is Ian Dolphin, the Executive Director of the Apereo Foundation, with the official pronunciation of the word \"Apereo\".'
+                                }
+                            }
+                        }
+                    ]
+                });
+        };
+
+        /**
          * Test that verifies the youtube processor and assures that metadata is retrieved/set.
          */
         it('verify youtube link processing works', function(callback) {
```

Comment:
Although the official end-of-life date was a while ago, the API has only recently defaulted all its responses.

From a travis build:
```
1) Preview processor Preview generation verify youtube link processing works:
      Uncaught AssertionError: "https://youtube.com/devicesupport" == "How to prounounce \"Apereo\""
      + expected - actual
      +"How to prounounce \"Apereo\""
      -"https://youtube.com/devicesupport"
      
      at /home/travis/build/oaeproject/Hilary/node_modules/oae-preview-processor/tests/test-previews.js:925:24
      at /home/travis/build/oaeproject/Hilary/node_modules/oae-preview-processor/tests/test-previews.js:262:29
      at Request._callback (/home/travis/build/oaeproject/Hilary/node_modules/oae-rest/lib/util.js:9:8822)
      at Request.self.callback (/home/travis/build/oaeproject/Hilary/node_modules/oae-rest/node_modules/request/request.js:121:22)
      at Request.emit (events.js:98:17)
      at Request.<anonymous> (/home/travis/build/oaeproject/Hilary/node_modules/oae-rest/node_modules/request/request.js:978:14)
      at Request.emit (events.js:117:20)
      at IncomingMessage.<anonymous> (/home/travis/build/oaeproject/Hilary/node_modules/oae-rest/node_modules/request/request.js:929:12)
      at IncomingMessage.emit (events.js:117:20)
      at _stream_readable.js:938:16
```